### PR TITLE
commit on api contract swagger set up

### PR DESCRIPTION
**Pull request by zircadia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,18 +10,22 @@
     "type": "git",
     "url": ""
   },
-  "author": {"name":"Kate Clark","name2":"Jen Lebaron","name3":"Dana McKeen","name4":"Hailey Mundt",
-    "name5":"Jamie Peck","name6":"Jennifer Thomas"},
```
Comment (zircadia) [inline package.json:13]: Why did you change the authors?

File: app.js [hunk 2]
```diff
@@ -57,7 +82,9 @@ app.use((error, req, res, next ) => {
     res.status(status).json({message: message});
 })
 
-mongoose.connect(MONGODB_URL)
-    .then(result => {
-app.listen(8080)})
-    .catch(err => console.log(err))
```
Comment (zircadia) [inline app.js:63]: Why are we losing our mongoose and mongodb connection?  don't we need to connect still to mongodb? Or did you just comment it out so you could work with swagger until the db is setup?

**Review by zircadia — COMMENTED** [review]
I have many questions! haahaha If you would take a look and let me know about my questions on these changes, I would appreciate that! Thanks!

File: package.json (hunk 1 above)
Comment (haileymm) [reply package.json:13]: Hmm... I didn't change authors directly and I don't know what I could have done to change it?

File: app.js (hunk 2 above)
Comment (haileymm) [reply app.js:63]: Yes just commented it out so I could get swagger working until we get the db set up.

File: package.json (hunk 1 above)
Comment (haileymm) [reply package.json:13]: Would it do that when I install dependencies? I don't know...

Comment (haileymm) [reply package.json:13]: I added them back. Should be good now.
